Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/fragment/SubmissionResultPositiveOtherWarningFragment.kt

```diff
@@ -120,30 +110,10 @@ class SubmissionResultPositiveOtherWarningFragment :
             initiateWarningOthers()
         }
         binding.submissionPositiveOtherWarningHeader.headerButtonBack.buttonIcon.setOnClickListener {
-            handleSubmissionCancellation()
+            activity?.onBackPressed()
```

Comment:
Not sure if this is the proper way to handle backwards navigation here, usage between this implementation and an additional navigation path differs in the app.
Any input on which way to choose is appreciated :+1:

Reply:
does "findNavController().popBackStack()" do the trick? Otherwise this seems fine